aiven/client/client.py

```diff
@@ -175,7 +175,7 @@ def verify(
                 time.sleep(0.2)
 
         # Check API is actually returning data or not
-        if response.status_code == HTTPStatus.NO_CONTENT or response.headers.get("Content-Length", "0") == "0":
+        if response.status_code == HTTPStatus.NO_CONTENT or len(response.content) == 0:
```

Comment:
content is always read just below so I guess this doesn't really introduce any performance penalty compared to the previous implementation.